### DISCUSSION TITLE
Embed Music Assistant into dashboard

### DISCUSSION THREAD

**Discussion by timmelander** [music-assistant/support#749] [Feature requests and ideas]
Currently Music Assistant upon installation creates its own dashboard, but that is not flexible. Please provide a way to embed Music Assistant into a card that could be added to an existing dashboard. 

I found a forum saying use a webpage card with url: https://[ha-base-url]/lib/mass, but that prompts a popup asking for a hostname when using iOS.

**Comment by OzGav** [comment]
Sorry Marcel advised that he thinks he has fixed this in 2022.7.2 and asked you to continue the discussion in the existing thread. Its hard to keep track of things if we have mutiple. I will lock this now and please continue the discussion in https://github.com/music-assistant/hass-music-assistant/discussions/594
